.example.env

```diff
@@ -0,0 +1,3 @@
+SECRET_KEY=notSoSecretKLCyNUgnY1lrcCAPHQvrKRRjGeq9HylDGx48j1kzntgtl8to7kzEoXfjgW72eboc7X6ho
```

Comment:
The name of the file seems to be `.example.env` while on the Readme and other tutorials we have `.env.example`. It's probably good to keep it consisten, what do you think?